scripts/envsetup.sh

```diff
@@ -35,23 +39,21 @@ fi
 METADIR=${METADIR:-${SOURCEDIR}/../..}
 
 if [[ ! -f "${BUILDDIR}/conf/local.conf" ]]; then
-  declare -a DISTRO_CONFIGS=${supported_distros[$DISTRO]}
-  [[ -n ${DISTRO_CONFIGS[@]} ]] && { echo "Using (${DISTRO_CONFIGS[*]}) for the specified distro '$DISTRO'"; } || { echo "The specified distro $DISTRO is not supported"; return 1; }
-
```

Comment:
While this sort of one-line if-replacement logic is a bit awkward to me, do you think the check against supported distros is not worth keeping?

Reply:
You're right, I've put a check back.